src/components/utils/downloadUtils.ts

```diff
@@ -106,32 +107,27 @@ export function useDownloadUtils() {
             fileName?: string
         ): Promise<void> => {
             try {
-                const result = await fetchConvertedCase(
+                const exportUuid = await fetchConvertedCase(
                     caseElement.elementUuid,
                     fileName || caseElement.elementName, // if no fileName is provided or empty, the case name will be used
                     format,
                     formatParameters,
                     abortController2
                 );
 
-                let downloadFileName =
-                    result.headers.get('Content-Disposition')?.split('filename=')[1] ??
-                    fileName ??
-                    caseElement.elementName;
-                // We remove quotes
-                downloadFileName = downloadFileName.substring(1, downloadFileName.length - 1);
-
-                const blob = await result.blob();
-
-                triggerDownload({ blob, filename: downloadFileName });
+                const identifier = buildExportIdentifier(exportUuid);
+                setExportSubscription(identifier);
+                snackInfo({
+                    messageTxt: intl.formatMessage({ id: 'export.message.started' }),
+                });
             } catch (error: any) {
                 if (error.name === 'AbortError') {
                     throw error;
                 }
-                handleDownloadError(caseElement, error);
+                handleDownloadError(caseElement, error.message || 'Export failed');
```

Comment:
```suggestion
                handleDownloadError(caseElement, error.message ?? String(error));
```
Do not hard code a none-translated text directly.

Reply:
done